### ISSUE TITLE
About train progress

### ISSUE THREAD

**Issue by nalibjchn** [simontomaskarlsson/CycleGAN-Keras#13]
Hello @simontomaskarlsson,

I is training cycleGAN, I used horse2zebra dataset for training, my epoch = 200, and my batch-size =1,  I use patchGan for the discriminator.
As your experience, how long will I train can see the result clearly? 
please see the screenshot which is a part of a result I trained, but it looks the translated image is the same as the original image.  I am not sure whether my mode TEMPhas something wrong or others.
could you give me some suggestion? 

epoch_index: 3/200][batch_index:550/1067] [D loss: 0.140700, acc:  54%] [G loss: 4.895703, adv: 0.535407, recon: 0.174021, id: 0.180061]
![image](https://user-images.githubusercontent.com/44066194/62067385-7ce17900-b22b-11e9-97f5-bb199daa7845.png)

by the way, I use google colab GPU for training.
Thank you in advance

**Comment by simontomaskarlsson** [comment]
Hi @lindalibjchn,

How many epochs it takes for the generators to learn the translation good enough to see some kind of result depends a lot on the specific image domains, i.e. A and B. From my experience you could usually see some results after about 50 epochs, and after 100 only minor details where improved by the generators.

If I understand it correctly your examples are after three epochs (?) and my guess is that more training is needed.

**Comment by VirtualAI** [comment]
Hello,
what options to lead to best results? 
Do you recommend something out of

-dentity learning (on different modulus of training iterations)
-PatchGAN in discriminators
-Multi-scale discriminators
-Resize convolution in generators

**Comment by nalibjchn** [comment]
I am using two options according to the original paper.
- identity learning
- PatchGAN in discriminators
- reflection padding

**Comment by simontomaskarlsson** [comment]
Hi @VirtualAI,

Using the settings as @lindalibjchn is a good choice I think.

From my research I found that identity learning helped with preserving for example colors of cars but could also leed to other problems if applied too much. As for resize convolution it did not effect the results in any major way.

PatchGAN in discriminators will almost always improve the results I believe.
